Dockerfile

```diff
@@ -39,6 +40,7 @@ COPY --from=builder /opt/app-root/bin/beyla .
 COPY --from=builder /opt/app-root/LICENSE .
 COPY --from=builder /opt/app-root/NOTICE .
 COPY --from=builder /opt/app-root/third_party_licenses.csv .
+COPY --from=builder /opt/app-root/opentelemetry-javaagent.jar .
```

Comment:
As something to improve in the future, we should find a way track the javaagent version and include it in our vulnerability scans.